eclipse-scout-core/src/form/fields/smartfield/SmartField.js

```diff
@@ -8,7 +8,24 @@
  * Contributors:
  *     BSI Business Systems Integration AG - initial API and implementation
  */
-import {arrays, Device, fields, FormField, InputFieldKeyStrokeContext, keys, LookupCall, objects, QueryBy, scout, SimpleLoadingSupport, SmartFieldCancelKeyStroke, SmartFieldLayout, Status, strings, ValueField} from '../../../index';
```

Comment:
@cguglielmo Is this (new) formatting of imports ok or should i revert to the old style?

Reply:
it's ok